src/main/java/gr8pefish/ironbackpacks/container/ContainerBackpack.java

```diff
@@ -57,12 +61,29 @@ public ContainerBackpack(@Nonnull ItemStack backpackStack, @Nonnull InventoryPla
         this.backpackSize = backpackInfo.getVariant().getBackpackSize();
 
         setupSlots(inventoryPlayer, itemHandler, hand);
+
+        for (Slot slot : inventorySlots) {
```

Comment:
Can this check not be done when creating the slot?

Reply:
It could but... the OOP purist in me screaming "NO! How dare you!". Adding this logic to the inventory slot setup loop seems like too much coupling for my taste. It's O(n) either way.

I moved it to `setupSlots()` at least.